src/app/mapControls/Print.js

```diff
@@ -98,11 +100,54 @@ define([
 
             this.inherited(arguments);
         },
+        setRequestPreCallback: function (patch) {
+            // summary:
+            //      set the function to inspect the print request
+            // patch: bool whether to patch or noop the precallback
+            console.log('app.mapControls.Print:setRequestPreCallback', arguments);
+
+            var precallback = undefined;
```

Comment:
`= undefined;` is redundant here. `var precallback;` is the same thing isn't it?